### PR TITLE
Set up CI

### PR DESCRIPTION
**Pull request by erbridge**
This adds in functionality to enforce 14 day review times (replacing the non-functional implementation that already existed with a version that uses labels for transparency). An improvement to this would be for it to ignore PRs labeled as "dependencies" or "tooling" so we can get those things merged quickly, but removing the "new" label allows you to work around it in the meantime.

It also adds in CI for formatting, with the same autofix behaviour as the Playbook. We switch to Prettier instead of Remark for formatting and auto-commit any formatting fixes it can make on a PR. We still use Remark to check that links work, but don't allow it to attempt to fix anything (since it can't).

### CODE REVIEW

File: .github/workflows/lint-and-fix.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  pull_request:
+
+jobs:
+  lint-and-fix:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: nodenv/actions-setup-nodenv@v2.0.4
```
Comment (CristinaRO) [inline .github/workflows/lint-and-fix.yml:11]: Any specific reason to try and use v2.0.4, or can we try the latest version that might solve the issue? (If the issue is similar to https://github.com/nodenv/actions/issues/13#issuecomment-749563088 )

Comment (erbridge) [reply .github/workflows/lint-and-fix.yml:11]: I _think_ that's what the GitHub Marketplace told me was the latest version. So no, we should use a later version.

Comment (erbridge) [reply .github/workflows/lint-and-fix.yml]: Fixed by using a different thing.

File: .github/workflows/lint-and-fix.yml [hunk 2]
```diff
@@ -9,6 +9,8 @@ jobs:
       - uses: actions/checkout@v2
 
       - uses: nodenv/actions-setup-nodenv@v2.0.4
+        env:
+          ACTIONS_ALLOW_UNSECURE_COMMANDS: true
```
[the remark this answers is not in the document]
Comment (erbridge) [reply .github/workflows/lint-and-fix.yml]: Stripped it out.

**Review by mec — APPROVED** [review]
Lots here! I was able to follow along as best I could and nothing jumped out at me, as this is tooling I am happy to hit approve.

File: .github/workflows/review-requirements/update-all-prs.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+on:
+  schedule:
+    # Every 10 minutes
+    cron: "*/10 * * * *"
```
Comment (jacksonj04) [inline .github/workflows/review-requirements/update-all-prs.yml:4]: Is this a bit aggressive given the volume we actually deal with? Maybe every hour (or even two) would be enough, given it's unlikely we'll have a PR waiting to merge the very moment it's _technically_ had enough time.

**Review by jacksonj04 — APPROVED** [review]
One tiny bit on the frequency of the check for tags, but otherwise 👍🏼

File: .github/workflows/review-requirements/update-all-prs.yml (hunk 3 above)
Comment (erbridge) [reply .github/workflows/review-requirements/update-all-prs.yml:4]: Yeah. I'll shift to every hour. Could probably also limit it to working hours (if I can work out the `cron` expression for that).

Comment (erbridge) [reply .github/workflows/review-requirements/update-all-prs.yml]: Done.

**Pull request merged**